logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -529,26 +529,45 @@ class TestPipeline < LogStash::Pipeline
     end
   end
 
-  context "when collecting metric in the pipeline" do
+  context "when collecting metrics in the pipeline" do
     subject { described_class.new(config, { :metric => metric, :pipeline_id => pipeline_id }) }
     let(:pipeline_id) { :main }
     let(:metric) { LogStash::Instrument::Metric.new }
     let(:number_of_events) { 1000 }
+    let(:multiline_id) { "my-multiline" }
+    let(:multiline_id_2) { "my-multiline_2" }
```

Comment:
can we use more meaningful names, not using `_2` as they don't really have meaning.